pkg/syncserver/sync_server.go

```diff
@@ -265,6 +290,22 @@ func (s *Server) serve(cxt context.Context) {
 			return
 		}
 
+		logCxt.Infof("Accepted from %s", conn.RemoteAddr())
+		tlsConn, ok := conn.(*tls.Conn)
```

Comment:
Well, you still need to be able to handle non-TLS connections when TLS is disabled.